docs/navigation-configuration.md

```diff
@@ -73,6 +73,8 @@ window.Luigi.setConfig({
 - **context** sends the specified object as context to the view. Use this parameter in combination with the dynamic **pathSegment** to receive the context through the context listeners of **Luigi client**. This is an alternative to using the dynamic value in the **viewUrl**.
 - **defaultChildNode** sets the child node that Luigi activates automatically if the current node has no **viewUrl** defined. Provide **pathSegment** of the child node you want to activate as a string.
 - **isolateView** renders the view in a new frame when you enter and leave the Node. This setting overrides the same-domain frame re-usage. The **isolateView** is disabled by default.
+- **loadingIndicator.enabled** shows a loading indicator when switching between micro front-ends. Super fast micro front-ends might want to disable this feature. The **loadingIndicator.loadingIndicator** is enabled by default.
```

Comment:
1. ...`when you switch between micro front-ends.`
2. "Super fast micro front-ends might want to disable this feature." -> Super fast micro front-ends are not humans so they cannot literally "want" anything. Please paraphrase.

Reply:
Something like "If you have a super fast micro frontend, you can disable this feature to prevent  loading indicator flickering."

Reply:
Just two corrections:
1. `front-end` as in the rest of the document
2.  `prevent loading indicator from flickering.` ?